src/PageLayout/PageLayout.tsx

```diff
@@ -675,9 +675,7 @@ const Pane = React.forwardRef<HTMLDivElement, React.PropsWithChildren<PageLayout
               ? ['100%', null, 'clamp(var(--pane-min-width), var(--pane-width), var(--pane-max-width))']
               : paneWidths[width],
             padding: SPACING_MAP[padding],
-            [`@media screen and (min-width: ${theme.breakpoints[1]})`]: {
-              overflow: 'auto',
-            },
+            overflow: [null, 'auto'],
```

Comment:
Ah, I think the correct breakpoint is actually this: 
```suggestion
            overflow: [null, null, 'auto'],
```